recipes/laspy/meta.yaml

```diff
@@ -0,0 +1,70 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+{% set name = "laspy" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "9561d6d4cfb5ebfa8201151415d45085" %}
+# sha256 is the prefered checksum -- you can get it for a file with:
+#  `openssl sha256 <file name>`.
+# You may need the openssl package, available on conda-forge
+#  `conda install openssl -c conda-forge``
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  # If the installation is complex, or different between Unix and Windows, use separate bld.bat and build.sh files instead of this key.
+  # By default, the package will be built for the Python versions supported by conda-forge and for all major OSs.
+  # Add the line "skip: True  # [py<35]" (for example) to limit to Python 3.5 and newer, or "skip: True  # [not win]" to limit to Windows.
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    # When setuptools is available add the `--single-version-externally-managed --record record.txt` above.
+    - setuptools
+    # if your project compiles code (such as a C extension) then add `toolchain` as a build requirement.
+  run:
+    - python
+    - numpy
+
+test:
+  imports:
+    - laspy
+    - laspy.file
+    - laspy.header
+    - laspy.base
+
+about:
+  home: https://github.com/laspy/laspy
+  # Remember to specify the license variants for BSD, Apache, GPL, and LGLP.
+  # Prefer the short version, e.g: GPL-2.0 instead of GNU General Public License version 2.0
+  # See https://opensource.org/licenses/alphabetical
+  license: BSD-3-Clause
+  # The license_family, i.e. "BSD" if license is "BSD-3-Clause". (optional)
+  license_family: BSD
+  # It is strongly encouraged to include a license file in the package,
+  # (even if the license doesn't require it) using the license_file entry.
+  # See http://conda.pydata.org/docs/building/meta-yaml.html#license-file
+  license_file: LICENSE.txt
```

Comment:
It looks like the `MANIFEST.in` doesn't specify the license file so it doesn't show up in the source distribution on pypi and will fail. See the following as a work-around:

http://conda-forge.github.io/docs/meta.html#packaging-the-license-manually